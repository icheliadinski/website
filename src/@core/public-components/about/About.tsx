import React from 'react'
import { SideCard } from '@core/ui-kit'
import { calculateAge } from '@core/services'
import styles from './About.module.scss'

export const About: React.FC = () => {
  const age = calculateAge(new Date(1994, 6, 18))
  return (
    <SideCard>
      <article className={styles.article}>
        <h1 className={styles.header}>about me</h1>
        <h2 className={styles.subinfo}>
          <span>{age} years</span>
          <span>New York</span>
          <span>Developer</span>
        </h2>
        <p>
          JavaScript/GoLang Developer with a Bachelorâ€™s Degree in Computer
          Science in Information Technology from Belarusian State University of
          Informatics and Radioelectronics and 5+ years of software development
          experience.
        </p>
        <p>Analytical thinker who can solve complicated problems quickly.</p>
      </article>
      <article className={styles.article}>
        <h2 className={styles.subheader}>my services</h2>
        <p>TBD</p>
      </article>
    </SideCard>
  )
}
