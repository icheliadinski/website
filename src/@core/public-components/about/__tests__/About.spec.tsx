import React from 'react'
import { shallow, ShallowWrapper } from 'enzyme'
import { About } from '../About'

describe('[About]', () => {
  let wrapper: ShallowWrapper

  beforeEach(() => {
    wrapper = shallow(<About />)
  })

  it('renders successfully', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
