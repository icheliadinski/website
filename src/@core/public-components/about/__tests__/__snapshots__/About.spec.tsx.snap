// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[About] renders successfully 1`] = `
<article>
  About Me
</article>
`;
